services/executionlayer/src/main/java/tech/pegasys/teku/services/executionlayer/ExecutionLayerService.java

```diff
@@ -193,6 +193,9 @@ private static ExecutionLayerManager createRealExecutionLayerManager(
 
     final ExecutionClientHandler executionClientHandler;
     if (config.isExchangeCapabilitiesEnabled()) {
+      LOG.info(
+          "Using Engine API method negotiation with Execution Client (engine_exchangeCapabilities)");
+
```

Comment:
perhaps lets humanise this?
`Negotiating interfaces to use with execution client.`

I don't think they care about exchangeCapabilities etc... Im not sure my wording is perfect either - thoughts?